src/index.js

```diff
@@ -43,7 +43,10 @@ class ReactPlayerLoader extends React.Component {
    */
   constructor(props) {
     super(props);
-    this.refNode = React.createRef();
```

Comment:
I wonder how easy it would be to feature detect the new ref feature and use that whenever possible and otherwise drop down to the old ref-support?

Reply:
There are polyfills available, such as https://www.npmjs.com/package/react-create-ref , but they seem to be overkill.  The callback ref method seems like it isn't going anywhere anytime soon: https://reactjs.org/docs/refs-and-the-dom.html#callback-refs

Reply:
that polyfill is actually not too bad. But yeah, supporting the old stuff for a while longer isn't bad.

Reply:
Yeah, I'd rather support 15 and 16 than 16.3 and some unknown future version that might break backward compatibility.